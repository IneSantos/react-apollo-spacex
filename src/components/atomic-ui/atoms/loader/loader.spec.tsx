import React from "react";
import { render } from "@testing-library/react";
import Loader from "./loader";

describe("Loader atom tests", () => {
  test(`Loader component render`, () => {
    const { container } = render(<Loader />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
