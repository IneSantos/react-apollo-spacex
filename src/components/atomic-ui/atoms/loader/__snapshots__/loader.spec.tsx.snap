// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader atom tests Loader component render 1`] = `
<div
  class="loder"
>
  <div
    class="lds-ellipsis"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
